### PR TITLE
 Added ⚡ content to `Getting Started` > `Hardware overview`

### PR DESCRIPTION
**Pull request by Bosch-0**
Adding Lightning content to the hardware overview page, solves https://github.com/BitcoinDesign/Guide/issues/466

Deploy: https://deploy-preview-476--sad-borg-390916.netlify.app/guide/getting-started/hardware/

### CODE REVIEW

File: guide/getting-started/hardware.md [hunk 1]
```diff
@@ -37,14 +37,11 @@ Illustration sources
 
 # Hardware overview
 
-Generally, any hardware device that is connected to the Internet can interact with the Bitcoin network. While personal computers and smartphones work great for general wallet management and transactions, some dedicated hardware products are optimized for the nuances of Bitcoin. We’ll explore them and how they work here.
+General purpose hardware like personal computers and smartphones work great for most basic Bitcoin operations. Though In some circumstances dedicated hardware devices are optimized for the nuances of Bitcoin. We’ll explore them and how they work here.
```
Comment (GBKS) [inline guide/getting-started/hardware.md:40]:
```suggestion
General purpose hardware, like personal computers and smartphones, work great for most basic Bitcoin operations. In some circumstances, dedicated hardware devices are optimized for the nuances of Bitcoin. We’ll explore them and how they work here. 
```

File: guide/getting-started/hardware.md [hunk 2]
```diff
@@ -56,9 +53,17 @@ Simple, cheap, and robust storage solutions for recovery phrases come in many fo
    height = 400
 %}
 
+Many users backup their bitcoin recovery phrases using metal plates that have their recovery phrase etched into them. This protects the recovery phrase from physical destruction from fire and/or water. [Seedplate](https://seedplate.com/) is an example of such a product.
+
+A more simple option is writing your recovery phrase on a piece of paper. Compared to steel paper can be easily destroyed making this option less secure.
```
Comment (GBKS) [inline guide/getting-started/hardware.md:58]:
```suggestion
A more simple option is writing your recovery phrase on a piece of paper. Compared to steel, paper can be easily destroyed, making this option less secure.
```

File: guide/getting-started/hardware.md [hunk 3]
```diff
@@ -71,14 +76,13 @@ You may already be familiar with key fobs from your bank or work. They are small
...
+Bitcoin hardware wallets, also known as external signers, act like Bitcoin centric security keys. They isolate the recovery phrase, private keys, and other sensitive data from the internet and other devices.
 
-Because hardware wallets are so optimized for this specific use case, they have tiny screens and limited input options. Most interactions with them happen via desktop software that the manufacturer provides. The application sends only specific requests to the hardware wallet to verify or sign data. This provides a strong balance of security and convenience.
+Hardware wallets only exchange non-sensitive information with external devices. Sensitive processes, such as signing a transaction to open a Lightning network payment channel, happen on the device. Most interactions with hardware wallets happen via desktop [software like wallets]({{ '/guide/getting-started/software/#wallets' | relative_url }}).
```
Comment (GBKS) [inline guide/getting-started/hardware.md:81]:
```suggestion
Hardware wallets only exchange non-sensitive information with external devices. Sensitive processes, such as signing a transaction to open a Lightning network payment channel, happen on the device. Most interactions with hardware wallets happen via desktop [software, like wallets]({{ '/guide/getting-started/software/#wallets' | relative_url }}). 
```

File: guide/getting-started/hardware.md [hunk 4]
```diff
@@ -90,13 +94,19 @@ Running your [own node]({{ '/guide/glossary/node/#self-hosted-node' | relative_u
    height = 400
 %}
 
-Dedicated node hardware is particularly relevant for heavy users of the Lightning network. While a bitcoin node can be turned off and on, Lightning nodes need to be online at all times to maintain payment channels with other nodes. Smartphones are typically online 24/7 but are not suited for this task due to large amounts of data transfer and storage capacity requirements.
+It is quite common to have dedicated hardware to run Bitcoin [node software]({{ '/guide/getting-started/software/#nodes' | relative_url }}). Running a Bitcoin node on dedicated hardware ensures your node stays up to sync with the network and is regularly verifying the network rules increasing Bitcoin’s security. Some node software also comes packaged with other third-party applications of which may benefit from dedicated hardware and more regular uptime.
```
Comment (GBKS) [inline guide/getting-started/hardware.md:97]:
```suggestion
It is quite common to have dedicated hardware to run Bitcoin [node software]({{ '/guide/getting-started/software/#nodes' | relative_url }}). Running a Bitcoin node on dedicated hardware ensures your node stays in sync with the network and is regularly verifying the network rules, increasing Bitcoin’s security. Some node software also comes packaged with other third-party applications which may benefit from dedicated hardware and more regular uptime. 
```

File: guide/getting-started/hardware.md [hunk 5]
```diff
@@ -90,13 +94,19 @@ Running your [own node]({{ '/guide/glossary/node/#self-hosted-node' | relative_u
...
 
-While [plug-and-play hardware nodes]({{ '/guide/glossary/node/#hardware-node' | relative_url }}) can solve various problems for bitcoin users, they have not seen widespread adoption as commercial products.
+Heavy users of the Lightning Network, such as routing node operators, often run Lightning nodes on dedicated hardware. Lightning nodes require 24/7 uptime, which is much easier to achieve with dedicated hardware. Lightning nodes also require the private keys to be stored on the same device. Dedicated hardware reduces the attack surface and makes it easier to secure these keys. 
 
-## Miners
+[Plug-and-play hardware nodes](https://samouraiwallet.com/nodl) exist and make it simple to set up a dedicated hardware node. The most common way people run dedicated hardware nodes is installing a node OS like [Umbrel](https://getumbrel.com/) or [MyNode](https://mynodebtc.com/) onto a hardware device.
```
Comment (GBKS) [inline guide/getting-started/hardware.md:103]:
```suggestion
[Plug-and-play hardware nodes](https://samouraiwallet.com/nodl) exist and make it simple to set up a dedicated hardware node. The most common way people run dedicated hardware nodes is installing a node OS, like [Umbrel](https://getumbrel.com/) or [MyNode](https://mynodebtc.com/), on a hardware device.
```

File: guide/getting-started/hardware.md [hunk 6]
```diff
@@ -90,13 +94,19 @@ Running your [own node]({{ '/guide/glossary/node/#self-hosted-node' | relative_u
...
 
-Since Bitcoin's inception a multi-billion dollar industry has emerged around the mining of new bitcoins. Today, dedicated hardware (so-called ASICS, or application-specific integrated circuits) are optimized for continuously executing the Bitcoin hashing algorithm (called [SHA-256](https://en.wikipedia.org/wiki/SHA-2){:target="_blank"}) over and over until an acceptable hash is found. A modern miner (in 2020) can generate around 14,000,000,000,000 hashes per second. The more hashes generated, the higher that miner’s chance of “finding” a new block and receiving newly minted bitcoin as a reward.
+See [nodes]({{ '/guide/glossary/node/' | relative_url }}) for more details.
+
+## Mining hardware
+
+Miners are the backbone of Bitcoin generating new blocks which creates new bitcoin. Dedicated mining hardware, which runs dedicated [mining software]({{ '/guide/getting-started/software/#mining' | relative_url }}), exists for mining bitcoin. Various billion dollar industries have developed around the manufacture and deployment of such hardware.
```
Comment (GBKS) [inline guide/getting-started/hardware.md:109]:
```suggestion
Miners are the backbone of Bitcoin generating new blocks, which creates new bitcoin. Dedicated mining hardware, which runs dedicated [mining software]({{ '/guide/getting-started/software/#mining' | relative_url }}), exists for mining bitcoin. Various billion dollar industries have developed around the manufacture and deployment of such hardware.
```

File: guide/getting-started/hardware.md [hunk 7]
```diff
@@ -124,6 +136,14 @@ A Bitcoin ATM (Automated Teller Machine) allows users to purchase or sell bitcoi
    height = 400
 %}
 
+Much like traditional ATMs, Bitcoin ATMs allow the deposit and withdrawal of money. Bitcoin ATMs however allow someone to deposit fiat currencies in exchange for bitcoin. 
+
+Modern Bitcoin ATMs take advantage of the Bitcoin Lightning network which enables almost instant withdrawals and cheaper fees making the purchase experience more friendly and cost effective.
```
Comment (GBKS) [inline guide/getting-started/hardware.md:141]:
```suggestion
Modern Bitcoin ATMs take advantage of the Bitcoin Lightning network. This enables almost instant withdrawals and cheaper fees, which makes the purchase experience more friendly and cost effective. 
```
Splitting up a very long sentence.

File: guide/getting-started/hardware.md [hunk 8]
```diff
@@ -124,6 +136,14 @@ A Bitcoin ATM (Automated Teller Machine) allows users to purchase or sell bitcoi
    height = 400
 %}
 
+Much like traditional ATMs, Bitcoin ATMs allow the deposit and withdrawal of money. Bitcoin ATMs however allow someone to deposit fiat currencies in exchange for bitcoin. 
+
+Modern Bitcoin ATMs take advantage of the Bitcoin Lightning network which enables almost instant withdrawals and cheaper fees making the purchase experience more friendly and cost effective. 
+
+Bitcoin ATMs are usually bound to local money transmitting laws and regulations, such as Know Your Customer (KYC) and anti-money laundering (AML) regulations. More on [Wikipedia](https://en.wikipedia.org/wiki/Bitcoin_ATM).
```
Comment (GBKS) [inline guide/getting-started/hardware.md:143]:
```suggestion
Bitcoin ATMs are usually bound to local money transmission laws and regulations, such as Know Your Customer (KYC) and Anti-Money Laundering (AML) regulations. More on [Wikipedia](https://en.wikipedia.org/wiki/Bitcoin_ATM).
```

**Comment by GBKS** [comment]
Looks great, this is a good revision. Just had a few minor nit-picky things, after that I think it's ready to go.

**Review by Bosch-0 — COMMENTED** [review]
Thanks for that :)

File: guide/getting-started/hardware.md [hunk 9]
```diff
@@ -56,9 +53,17 @@ Simple, cheap, and robust storage solutions for recovery phrases come in many fo
    height = 400
 %}
 
+Many users backup their bitcoin recovery phrases using metal plates that have their recovery phrase etched into them. This protects the recovery phrase from physical destruction from fire and/or water. [Seedplate](https://seedplate.com/) is an example of such a product.
```
Comment (GBKS) [inline guide/getting-started/hardware.md:56]:
```suggestion
Many users back up their wallet using metal plates that have their recovery phrase etched into them. This protects the recovery phrase from physical destruction from fire and  water. [Seedplate](https://seedplate.com/) is an example of such a product.
```

File: guide/getting-started/hardware.md [hunk 10]
```diff
@@ -71,14 +76,13 @@ You may already be familiar with key fobs from your bank or work. They are small
...
 
 ## Nodes
 
-Tens of thousands of computers participate in the Bitcoin network. They are called [nodes]({{ '/guide/glossary/node/' | relative_url }}). Nodes exchange, verify, and compare block and transaction data.
-Running your [own node]({{ '/guide/glossary/node/#self-hosted-node' | relative_url }}), instead of relying on a [third-party node]({{ '/guide/glossary/node/#third-party-node' | relative_url }}), lets you validate transactions on your own device. It also improves the privacy of your transaction and help secure the network. Several DIY and consumer hardware products make it easy to run your own node.
+A node is a device that participates in a network. In Bitcoin, there are two types of nodes to understand: A Bitcoin node which participates in the Bitcoin network, and a Lightning node which participates in the Lightning network. For a deeper dive into what purpose these nodes serve check out the [technology primer]({{ '/guide/getting-started/technology-primer/' | relative_url }}).
```
Comment (GBKS) [inline guide/getting-started/hardware.md:85]:
```suggestion
A node is a device that participates in a network. There are two types of nodes to understand: A Bitcoin node which participates in the Bitcoin network, and a Lightning node which participates in the Lightning network. For a deeper dive into what purpose these nodes serve check out the [technology primer]({{ '/guide/getting-started/technology-primer/' | relative_url }}).
```

File: guide/getting-started/hardware.md [hunk 11]
```diff
@@ -90,13 +94,19 @@ Running your [own node]({{ '/guide/glossary/node/#self-hosted-node' | relative_u
    height = 400
 %}
 
-Dedicated node hardware is particularly relevant for heavy users of the Lightning network. While a bitcoin node can be turned off and on, Lightning nodes need to be online at all times to maintain payment channels with other nodes. Smartphones are typically online 24/7 but are not suited for this task due to large amounts of data transfer and storage capacity requirements.
+It is quite common to have dedicated hardware to run Bitcoin [node software]({{ '/guide/getting-started/software/#nodes' | relative_url }}). Running a Bitcoin node on dedicated hardware ensures your node stays in sync with the network and is regularly verifying the network rules, increasing Bitcoin’s security. Some node software also comes packaged with other third-party applications which may benefit from dedicated hardware and more regular uptime.
```
Comment (GBKS) [inline guide/getting-started/hardware.md:97]:
```suggestion
It is quite common to have dedicated hardware to run Bitcoin [node software]({{ '/guide/getting-started/software/#nodes' | relative_url }}). This ensures your node stays in sync with the network and is regularly verifying the network rules, increasing Bitcoin’s security. Some node software also comes packaged with other third-party applications which may benefit from dedicated hardware and more regular uptime. 
```

File: guide/getting-started/hardware.md [hunk 12]
```diff
@@ -90,13 +94,19 @@ Running your [own node]({{ '/guide/glossary/node/#self-hosted-node' | relative_u
...
 
-While [plug-and-play hardware nodes]({{ '/guide/glossary/node/#hardware-node' | relative_url }}) can solve various problems for bitcoin users, they have not seen widespread adoption as commercial products.
+Heavy users of the Lightning Network, such as routing node operators, often run Lightning nodes on dedicated hardware. Lightning nodes require 24/7 uptime, which is much easier to achieve with dedicated hardware. Lightning nodes also require the private keys to be stored on the same device. Dedicated hardware reduces the attack surface and makes it easier to secure these keys. 
 
-## Miners
+[Plug-and-play hardware nodes](https://samouraiwallet.com/nodl) exist and make it simple to set up a dedicated hardware node. The most common way people run dedicated hardware nodes is installing a node OS, like [Umbrel](https://getumbrel.com/) or [MyNode](https://mynodebtc.com/), on a hardware device.
```
Comment (GBKS) [inline guide/getting-started/hardware.md:103]:
```suggestion
[Plug-and-play hardware nodes](https://samouraiwallet.com/nodl) make it simple to set up a dedicated hardware node. The most common way people run dedicated hardware nodes is installing a node OS, like [Umbrel](https://getumbrel.com/) or [MyNode](https://mynodebtc.com/), on a hardware device.
```

File: guide/getting-started/hardware.md [hunk 13]
```diff
@@ -108,11 +118,13 @@ Since Bitcoin's inception a multi-billion dollar industry has emerged around the
    height = 400
 %}
 
-Outside of purchasing hardware, the main cost of mining is electricity. That’s why mining facilities are found wherever electricity is abundant and cheap, such as in Iceland (geothermal) or in China (hydroelectricity).
+When Bitcoin was first launched back in 2009, mining hardware was mostly conducted by CPUs and GPUs. As the network grew in size, and the market around mining grew, more specialized and powerful hardware known as application-specific integrated circuits (ASICs) became the standard mining hardware. 
+
+Today, most Bitcoin miners are SHA-256 ASICs. SHA-256 ASICs which are optimized to continuously execute the SHA-256 hashing algorithm with the goal of finding a suitable hash and thus mining a new Bitcoin block. Modern SHA-256 ASIC miners (2021) are capable of producing 100 trillion SHA-256 hashes per second.
```
Comment (GBKS) [inline guide/getting-started/hardware.md:123]:
```suggestion
Today, most Bitcoin miners are SHA-256 ASICs. They are optimized to continuously execute the SHA-256 hashing algorithm with the goal of finding a suitable hash and thus mining a new Bitcoin block. Modern miners (2021) are capable of producing 100 trillion hashes per second.
```
Too much repetition of " SHA-256".

**Review by GBKS — CHANGES_REQUESTED** [review]
Had a few super minor tweaks after reading through the page again. Otherwise, ready to go.

File: guide/getting-started/hardware.md [hunk 14]
```diff
@@ -71,14 +76,13 @@ You may already be familiar with key fobs from your bank or work. They are small
    height = 400
 %}
 
-Bitcoin hardware wallets play a similar role but with several noteworthy differences. Like key fobs, they securely store an authentication method and isolate it from the Internet, networks, and other devices. In bitcoin, these are the seed, associated keys, and functions that require interaction with them.
+Bitcoin hardware wallets, also known as external signers, act like Bitcoin centric security keys. They isolate the recovery phrase, private keys, and other sensitive data from the internet and other devices.
```
Comment (pavlenex) [inline guide/getting-started/hardware.md:79]: This isn't entirely correct. Hardware wallets do not isolate sensitive data from the `internet`. As the matter a fact, majoirty of hardware wallets will force users to use third-party web-wallets that rely on their servers in order to send a transactions, leaking all sensitive information to the manufacturer. 

What most hardware wallets isolate is private key, and they just spit out the signature, which means they sign a transaction on a secure chip element, but any other sensitive data may be leaked.

For example xpub is in most cases leaked, and that's indeed a sensitive data.

And reading the paragraph below it, we may not even need this paragraph.

```suggestion
Bitcoin hardware wallets, also known as external signers, act like Bitcoin centric security keys. They isolate the recovery phrase and private keys from the internet and other devices.
```

File: guide/getting-started/hardware.md [hunk 15]
```diff
@@ -90,13 +94,19 @@ Running your [own node]({{ '/guide/glossary/node/#self-hosted-node' | relative_u
    height = 400
 %}
 
-Dedicated node hardware is particularly relevant for heavy users of the Lightning network. While a bitcoin node can be turned off and on, Lightning nodes need to be online at all times to maintain payment channels with other nodes. Smartphones are typically online 24/7 but are not suited for this task due to large amounts of data transfer and storage capacity requirements.
+It is quite common to have dedicated hardware to run Bitcoin [node software]({{ '/guide/getting-started/software/#nodes' | relative_url }}). Running a Bitcoin node on dedicated hardware ensures your node stays in sync with the network and is regularly verifying the network rules, increasing Bitcoin’s security. Some node software also comes packaged with other third-party applications which may benefit from dedicated hardware and more regular uptime. 
+
+Lightning node software can be run on a smartphone. Though, this often comes with trusting a third-party for certain node functions such as payment path construction. For this reason, dedicated hardware Lightning nodes may be a better option for those who do not want a trust-minimized setup.
```
Comment (pavlenex) [inline guide/getting-started/hardware.md:99]: > Lightning node software can be run on a smartphone.

This is a bold claim at this stage, is there a way to run a full lightning node on a smartphone? I don't think it is, though with LDK it should be possible. cc @moneyball ?

**Review by pavlenex — COMMENTED** [review]
Great work @Bosch-0, comparing it to the previous page, you managed to iron out quite a few things. I only have two concerns:

- A claim that a hardware wallet cannot leak sensitive data in one of the paragraphs, the paragraph explains this more accurately.
- A claim that you can run a lightning node on a smatphone, I am not really sure about this at this stage.

File: guide/getting-started/hardware.md (hunk 15 above)
Comment (GBKS) [reply guide/getting-started/hardware.md:99]: It doesn't state "full node", just "node software" in general, and then explains some of the tradeoffs. Looks accurate to me.

Comment (pavlenex) [reply guide/getting-started/hardware.md:99]: Node software assumes a node is run? You cannot run a node on a smartphone with any LN implementation afaik.

Comment (Bosch-0) [reply guide/getting-started/hardware.md:99]: You can run a Lightning node on a smartphone - Phoenix is an example: https://phoenix.acinq.co/faq#is-phoenix-a-real-lightning-node

Comment (pavlenex) [reply guide/getting-started/hardware.md:99]: Interesting, wasn't aware of this. I guess eclair is the only implementation that makes this possible, will have to dig how they do it technically.

**Review by pavlenex — APPROVED** [review]

**Review by pavlenex — APPROVED** [review]

**Comment by pavlenex** [comment]
@GBKS review and approve, can't merge since the `request a change` is blocking it

**Review by GBKS — APPROVED** [review]
Thanks for addressing all the feedback. Looks really good.

**Pull request merged**